README.md

```diff
@@ -1,96 +1,290 @@
-# Climate Change AI (CCAI) Backend [![CircleCI](https://circleci.com/gh/cc-ai/floods-backend.svg?style=svg)](https://circleci.com/gh/cc-ai/floods-backend)
+﻿CCAI Guide
 
-The CCAI project is an interdisciplinary project aimed at creating images of accurate, vivid, and personalized outcomes of climate change. Our goal is to use cutting-edge machine learning techniques to produce images of how neighborhoods and houses will look like following the effects of global warming. By creating a more visceral understanding of the effects of climate change, we aim to strengthen public support for necessary actions and motivate people to make impactful decisions. As a prototype, we first focus on modeling flood consequences on homes.
+-   [Welcome to Climate Change AI!](#welcome-to-climate-change-ai)
+-   [TECHNOLOGIES](#technologies)
```

Comment:
Can you title case the items in the ToC which are not title case? It seems like some are inconsistently caps?